hummingbot/connector/perpetual_trading.py

```diff
@@ -0,0 +1,83 @@
+from decimal import Decimal
+from typing import Dict, List
+from hummingbot.core.event.events import PositionMode, FundingInfo
+from hummingbot.connector.derivative.position import Position
+
+
+NaN = float("nan")
+s_decimal_NaN = Decimal("nan")
+s_decimal_0 = Decimal(0)
+
+
+class PerpetualTrading:
+    """
+    A base class (interface) that defines what perpetual trading is for Hummingbot.
+    """
+
+    def __init__(self):
+        super().__init__()
+        self._account_positions: List[Position] = []
```

Comment:
This should be a dictionary. Connectors that support Hedge-mode are going to have issues if it's a list. i.e. it'll be difficult to manage 2 contracts on  a particular market.

Reply:
Keeping it as a list will require the `get_position` function in dydx perp for instance to take an extra parameter to filter and return position side specific result.